arch/arm/include/stm32/stm32g47xxx_irq.h

```diff
@@ -0,0 +1,204 @@
+/****************************************************************************************************
+ *  arch/arm/include/stm32/stm32g47xxx_irq.h
+ *
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.  The
+ *  ASF licenses this file to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance with the
+ *  License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ *
+ ****************************************************************************************************/
+
+/* This file should never be included directly but, rather, only indirectly
+ * through nuttx/irq.h
+ */
+
+#ifndef __ARCH_ARM_INCLUDE_STM32_STM32G47XXX_IRQ_H
+#define __ARCH_ARM_INCLUDE_STM32_STM32G47XXX_IRQ_H
+
+/****************************************************************************************************
+ * Included Files
+ ****************************************************************************************************/
+
+#include <nuttx/config.h>
+#include <nuttx/irq.h>
+
+/****************************************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************************************/
+
+/* IRQ numbers.  The IRQ numbers correspond to the vector numbers and hence
+ * map directly to bits in the NVIC.  This does, however, waste several words
+ * of memory in the IRQ to handle mapping tables.
+ *
+ * Processor Exceptions (vectors 0-15) are common to all STM32 parts are are
+ * found in nuttx/arch/arm/include/stm32/irq.h.  They are not repeated here.
+ *
+ * Other interrupts (vectors >= 16) are defined below.
+ */
+
+#define STM32_IRQ_WWDG        (STM32_IRQ_FIRST+0)   /* 0:  Window Watchdog interrupt */
```

Comment:
Yes, it is. I think it is not checking for operator spacing in defines.

I did it this way because looking at other code, it seems that's how it is being done elsewhere.

WDYT?

Reply:
I went ahead and fixed operator spacing. All binary operators are surrounded by a single space per the coding standard.